src/Tokenizers/PHP.php

```diff
@@ -945,6 +946,56 @@ protected function tokenize($string)
                 continue;
             }//end if
 
+            /*
+                "readonly" keyword support
+                PHP < 8.1: Converts T_STRING to T_READONLY - Name tokens (T_NAME_*) have to be converted to T_STRING first
```

Comment:
Note: If I read this correctly, this means that this PR can't be forward-ported to the 4.x branch without changes. This may need some extra attention.

Reply:
Yes, I currently expend there are no `T_NAME_*` tokens.

Reply:
Rewritted.